vitest.config.ts

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
       reporter: ["text", "html", "json-summary"],
       lines: 50,
       branches: 50,
-      functions: 50,
```

Comment:
I have updated this to 45 due to the Mantine changes; in the future, I have plans to create a testing test coverage strategy.